README.md

```diff
@@ -85,6 +86,19 @@ pip install dohq-artifactory==0.5.dev243
 ```
 
 # Usage
+## Enable logging 
+For purpose of debugging or need of additional information you can configure logger to work with current library
```

Comment:
The sentence looks like we're describing what is logging for. We shouldn't do this in library documentation.

Good example in stripe https://github.com/stripe/stripe-python#logging
Short, contains some library-specific notes, talk about their logger only

Reply:
done